scripts/nightly_bench.py

```
@@ -0,0 +1,535 @@
+    skipped: list[dict] = []
+
+    headers = "  ".join(f"{label:>10}" for _, label, _, _ in CONFIGS)
+    log(f"    {'Benchmark':<40} {headers}")
+    log("    " + "─" * (40 + len(CONFIGS) * 12))
+
+    for path in benchmarks:
+        name = path.relative_to(TEST_DIR).as_posix()
+        req = requires_proofs(path)
+
+        # Qualification probes: the single-thread standard run and the proof run.
+        std_probe = None if req else probe(path, 1, False)
```

Ruff (0.15.18)

[warning] Boolean positional value in function call

(FBT003)